rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1657,7 +1657,7 @@ private void processSigning(
      */
     public byte[] getStateForBtcReleaseClient() throws IOException {
```

Comment:
Not related to this PR, but looks like we are going to need some tests for this method